www/front_src/src/Resources/ActionButton.tsx

```diff
@@ -34,7 +33,6 @@ const ActionButton = ({
         <IconButton
           className={classes.button}
           color="primary"
-          onClick={onClick}
           disabled={disabled}
```

Comment:
Maybe we could remove `disabled` because it is forwarded by `...prop`, isn't it ?